lib/records/fhir.rb

```diff
@@ -2,6 +2,7 @@ module Synthea
   module Output
     module FhirRecord
       SHR_EXT = 'http://standardhealthrecord.org/fhir/StructureDefinition/'.freeze
+      SYNTHEA_EXT = 'http://synthetichealth.github.io/synthea/'.freeze
```

Comment:
I used this for a previous implementation to add the values into the fhir record, but it is no longer used.  I will remove them.